types/tools/tune-events.d.ts

```diff
@@ -0,0 +1,4 @@
+export type MoveEvent = {
```

Comment:
This is relevant not only for tunes, so `hook-events` is better name for file I guess